Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM node:12-alpine
+FROM node:12
 
 WORKDIR /app
-COPY . .
-
-RUN npm --version && node --version
+COPY . . 
 
 RUN npm install && \
```

Comment:
so wird das nur in development funktionieren, npm install mit node_env production installiert keine devDependencies.
Eigentlich sollte die node_env also direkt nach npm install gesetzt werden, damit sie dann für build und die scripts gilt.
Default sollte production sein, sonst machen wir unsere installationen unsicher

Reply:
Ich glaube dann müssen wir aber schauen dass wir dei dev dependencies an der stelle zu normalen dependencies machen wo sie für den produktiven betrieb notwendig sind oder?